genienlp/server.py

```diff
@@ -154,7 +154,8 @@ def _init_request(self, request):
                 self.model.config, self.args.pretrained_model, args.src_locale, args.tgt_locale
             )
             self.numericalizer.update_language_dependent_properties(src_locale, tgt_locale)
-            self.model.update_language_dependent_configs(tgt_locale)
+            if hasattr(self.model, 'update_language_dependent_configs'):
```

Comment:
Rather than this, it would be cleaner to add an empty implementation of the method in the base class.